src/xtt.c

```diff
@@ -107,16 +107,9 @@ enftun_xtt_handshake(const char** server_hosts,
                      struct enftun_xtt* xtt)
```

Comment:
@dberliner This commit updates the code for XTT handshakes to support both SCM and TCP, but doesn't update `enftun.c` for the main TLS tunnel.